web/modules/custom/rdf_etl/src/Plugin/EtlProcessStep/ManualUploadStep.php

```diff
@@ -16,13 +22,54 @@
  *  label = @Translation("Manual upload"),
  * )
  */
-class ManualUploadStep extends ProcessStepBase implements PluginFormInterface {
+class ManualUploadStep extends ProcessStepBase implements PluginFormInterface, ContainerFactoryPluginInterface {
+
+  const SINK_GRAPH = 'http://etl-sink/';
+
+  protected $graphStore;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('sparql_endpoint')
+    );
+  }
+
+  /**
+   * ManualUploadStep constructor.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\rdf_entity\Database\Driver\sparql\Connection $sparql_connection
+   *   The SPARQL database connection.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, Connection $sparql_connection) {
+    $this->sparqlConnection = $sparql_connection;
+    $connection_options = $sparql_connection->getConnectionOptions();
+    $connect_string = 'http://' . $connection_options['host'] . ':' . $connection_options['port'] . '/sparql-graph-crud';
+    // Use a local SPARQL 1.1 Graph Store.
+    $this->graphStore = new GraphStore($connect_string);
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+  }
 
   /**
    * {@inheritdoc}
    */
   public function execute(array $data): void {
-    // TODO: Implement execute() method.
+    /** @var \EasyRdf\Http\Response $response */
+    $response = $this->graphStore->replace($data['graph'], self::SINK_GRAPH);
```

Comment:
As we've discussed on chat, we need to store the sink graph value in the plugin default configuration so that we're able to swap it in tests. That would mean the contents of `\Drupal\rdf_etl\ProcessStepBase::defaultConfiguration` will be:
```php
public function defaultConfiguration(): array {
  return [
    'sink_graph' => static::SINK_GRAPH,
  ];
}
```
`const SINK_GRAPH = 'http://etl-sink/';` declaration should be moved in the plugin interface and this line will become:
```php
$response = $this->graphStore->replace($data['graph'], $this->getConfiguration()['sink_graph']);
```

In this way we're able to create a process step plugin instance that uses a different graph:
```php
$config = ['sink_graph' => 'http://example.com/testing_sink_graph'];
$step = \Drupal::service('plugin.manager.etl_process_step')
  ->createInstance('convert_to_adms2', $config);
```